packages/flame_sprite_fusion/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+* TODO: Describe initial release.
```

Comment:
Melos will generate this

Reply:
So, should I remove the whole file or just the entry?

Reply:
Either leave it completely empty or remove it I think